src/MicrosoftTeams.ts

```diff
@@ -900,25 +908,20 @@ namespace microsoftTeams
          * The current UI theme the user is using.
          */
         theme?: string;
-
-        /**
-         * The context passed in as part of a deep link navigation to this page which should be used
-         * to restore a specific page state.
-         */
-        deepLinkContext?: string;
     }
 
     export interface DeepLinkParameters
     {
         /**
-         * Any context the page might need to restore a specific state for the user.
+         * The developer-defined unique id for the sub-entity this deep link points to. The entityId of this content will automatically be included
+         * so this field may be used to restore to a specific state within an entity, for example scrolling to or activating a specific piece of content.
          */
-        deepLinkContext?: string;
+        subEntityId?: string;
 
         /**
          * The label which should be displayed for this deep link when the link is rendered in a client.
          */
-        label?: string;
+        label: string;
```

Comment:
I wonder if this should be subEntityLabel and subEntityWebUrl?  Or if the interface should be SubEntityDeepLinkParameters?  To avoid confusion elsewhere with the main (pinned) entity's label and webUrl.